scheduleit/views/invites/show.php

```diff
@@ -53,6 +53,9 @@
 
             // Check for file to upload
             if (!empty($_FILES['file']['name'])) {
+                // delete old file first
+                $file_name = UPLOADS_ABSPATH . $meeting['hash'] . '/' . $_SESSION['user_onid'] . '_upload.*';
```

Comment:
This should already be handled here: https://github.com/OregonStateUniversity/MyEventBoard/blob/0ea5f910c9f0bab892c86a80036210d8df0e8a82/scheduleit/lib/file_upload.php#L95. Unless you've seen an issue?

Reply:
its possible my workspace didn't have this but neither edit nor he invite page were deleting previous upload files with a different ext.

Reply:
yeah i just didn't have the newest version of file_upload.php